testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/rx/RxCompletionStageProxyServerAsyncTest.java

```diff
@@ -0,0 +1,270 @@
+      client.close();
+   }
+
+   //////////////////////////////////////////////////////////////////////////////
+   
+   @Test
+   public void testGet() throws Exception {
+      String s = proxy.get();
+      Assert.assertEquals("x", s);
+   }
+
+   @Test
+   public void testGetThing() throws Exception {
+      Thing t = proxy.getThing();
+      Assert.assertEquals(new Thing("x"), t);
+   }
+
+   @Test
+   public void testGetThingList() throws Exception {
+      List<Thing> list = proxy.getThingList();
+      Assert.assertEquals(xThingList, list);
+   }
+
+   @Test
+   public void testPut() throws Exception {
+      String s = proxy.put("a");
+      Assert.assertEquals("a", s);
+   }
+
+   @Test
+   public void testPutThing() throws Exception {
+      Thing t = proxy.putThing("a");
+      Assert.assertEquals(new Thing("a"), t);
+   }
+
+   @Test
+   public void testPutThingList() throws Exception {
+      List<Thing> list = proxy.putThingList("a");
+      Assert.assertEquals(aThingList, list);
+   }
+
+   @Test
+   public void testPost() throws Exception {
+      String s = proxy.post("a");
+      Assert.assertEquals("a", s);
+   }
+
+   @Test
+   public void testPostThing() throws Exception {
+      Thing t = proxy.postThing("a");
+      Assert.assertEquals(new Thing("a"), t);
+   }
+
+   @Test
+   public void testPostThingList() throws Exception {
+      List<Thing> list = proxy.postThingList("a");
+      Assert.assertEquals(aThingList, list);
+   }
+
+   @Test
+   public void testDelete() throws Exception {
+      String s = proxy.delete();
+      Assert.assertEquals("x", s);
+   }
+
+   @Test
+   public void testDeleteThing() throws Exception {
+      Thing t = proxy.deleteThing();
+      Assert.assertEquals(new Thing("x"), t);
+   }
+
+   @Test
+   public void testDeleteThingList() throws Exception {
+      List<Thing> list = proxy.deleteThingList();
+      Assert.assertEquals(xThingList, list);
+   }
+
+   @Test
+   public void testHead() throws Exception {
+      try {
+         proxy.head();
+      } catch (Exception e) {
+         Assert.assertTrue(throwableContains(e, "Input stream was empty, there is no entity"));
+      }
+   }
+
+   @Test
+   public void testOptions() throws Exception {
+      String s = proxy.options();
+      Assert.assertEquals("x", s);
+   }
+
+   @Test
+   public void testOptionsThing() throws Exception {
+      Thing t = proxy.optionsThing();
+      Assert.assertEquals(new Thing("x"), t);
+   }
+
+   @Test
+   public void testOptionsThingList() throws Exception {
+      List<Thing> list = proxy.optionsThingList();
+      Assert.assertEquals(xThingList, list);
+   }
+   
+//   @Test
+   public void testUnhandledException() throws Exception {
+      try {
+         proxy.getThing();
+         Assert.fail("expecting Exception");
+      } catch (Exception e) {
+         Assert.assertTrue(e.getMessage().contains("500"));
+      }
+   }
+
+//   @Test
+   public void testHandledException() throws Exception {
```

Comment:
Is the commented out "@Test" here on purpose or a mistake?

Reply:
@asoldano, wow, looks like some unfinished cut-and-pasting. Totally missed it. Thanks!